src/api/ExamManagement.ts

```diff
@@ -0,0 +1,121 @@
+import { AxiosError, AxiosResponse } from "axios";
+import UnsignedProposalMessage from "./api_models/common/UnsignedProposalMessage";
+import APIError from "./api_models/errors/APIError";
+import Exam from "./api_models/exam_management/Exam";
+import handleAuthenticationError from "./AuthenticationHelper";
+import CommonHyperledger from "./CommonHyperledger";
+import APIResponse from "./helpers/models/APIResponse";
+
+export default class ExamManagement extends CommonHyperledger {
+    protected static endpoint = "/exam-management";
+    protected static serviceIdentifier = "exam";
+
+    constructor() {
+        super(ExamManagement.endpoint, ExamManagement.serviceIdentifier);
+    }
+
+    static async getVersion(): Promise<string> {
+        return super.getVersion();
+    }
+
+    async getExams(
+        examIds?: string[],
+        courseIds?: string[],
+        lecturerIds?: string[],
+        moduleIds?: string[],
+        types?: string[],
+        admittableAt?: Date,
+        droppableAt?: Date
```

Comment:
should be `admittableUntil`? (same for drop)

Reply:
No, this is a distinction in the API, as we do not want a direct match of data (admittableUntil as parameter would suggest filtering for exams with exact this value), but we actually want a "give me all exams that are admittable at a timestamp, i.e. with admittableUntil >= admittableAt